### PR TITLE
Use Go 1.11 with Modules support

### PR DESCRIPTION
**Pull request by rgo3**
With this commit we upgrade the CLI to use Go 1.11

### CODE REVIEW

**Comment by ArmandGrillet** [comment]
Is the `vendor/` directory still necessary?

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

**Comment by rgo3** [comment]
run all integration tests

File: .travis.yml [hunk 1]
```diff
@@ -4,14 +4,15 @@ os:
...
 
 go_import_path: github.com/dcos/dcos-cli
 
 env:
- - NO_DOCKER=1
+ - NO_DOCKER=1 GO111MODULE=on
 
-before_script:
+before_install:
+ - pwd
```
Comment (bamarni) [inline .travis.yml]: leftover from our tests

**Comment by rgo3** [comment]
@mesosphere-ci run integration tests

**Pull request merged**